tools/cloud-build/daily-tests/builds/ctk-dockerfile.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+tags: [dockerfile, m.pre-existing-vpc, m.vm-instance, vm]
+
+timeout: 3600s  # 1hr
+steps:
+## uses a pre-built Google Cloud image containing Docker CLI tool.
+- id: build-docker-image
+  name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcluster', 'tools/cloud-build/images/cluster-toolkit-dockerfile/']
+
+- id: create-deployment-folder
+  name: 'gcluster'
+  args:
+  - 'create'
+  - 'tools/cloud-build/daily-tests/blueprints/e2e.yaml'
+  - '--vars'
+  - 'project_id=$PROJECT_ID'
```

Comment:
You've probably already tested this, but just wanted to double-check: does the `$PROJECT_ID` variable get replaced with the correct value?

Reply:
Yes, it's replaced with the correct value. Cloud build automatically provides default substitutions for certain variables. `$PROJECT_ID` is one of them. By default `$PROJECT_ID`is given the ID of the Cloud project the build runs in. More details [here](https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values)

Reply:
Thanks for the explanation and link!